bin/sqlline.py

```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env python
+"""sqlline.py"""
```

Comment:
Is this a convention I don't know about? From a licensing perspective, I think it would be best to make sure the ASLv2 header be the first thing after the shebang.